src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -85,6 +85,12 @@ def _ensure_monitor_with_config(
     validated_config = validator.validated_data
     created = False
 
+    if (
+        features.has("organizations:crons-disable-new-projects", project.organization)
+        and not project.flags.has_cron_monitors
+    ):
+        return monitor
```

Comment:
This is options backed right?

Reply:
yep options backed in getsentry https://github.com/getsentry/getsentry/pull/12460/files#diff-90b033150d810eb0e7493b6d9712f7f5ae6bf5725b3c3ffb08931859677a14beR1891, flag declared in sentry